image/manifest.go

```diff
@@ -240,3 +244,32 @@ loop:
 	}
 	return nil
 }
+
+// ValidateManifestPatten validate the manifest's internal
```

Comment:
`Patten`?  Is the idea here to separate blob-only validation from blob+ancestors validation?  In that case, I think the names should be something like `ValidateManifestBlob` and `ValidateManifestAncestors` with a `ValidateManifest` that calls both of those parts.